examples/ess-human-presence-detector/ess-human-presence-detector.ino

```diff
@@ -0,0 +1,67 @@
+// this is a minimal version without error checking
+// this is meant purely to exaplain the API, and its use is not recommended
+// to use this in a serious application
```

Comment:
wording: its use is not recommended to use this -> usage is not recommended in

Reply:
heh, this is in fact copied from the ess-minimal example. will fix here and propagate